platform/android/java/lib/src/org/godotengine/godot/plugin/GodotPlugin.java

```diff
@@ -248,6 +265,22 @@ protected void runOnGLThread(Runnable action) {
 	 */
 	private native void nativeRegisterMethod(String p_sname, String p_name, String p_ret, String[] p_params);
 
+	/**
+	 * Used to complete registration of the {@link GodotPlugin} instance's methods.
```

Comment:
For both methods, can you update the naming scheme to be more explicit and match the Java naming style (e.g: pluginName, signalName, etc).

Reply:
Also make the two methods `static`, that should solve the crashing issue you're seeing.

Reply:
I made them static in 3.2 branch. It does'n help.